refmt.go

```diff
@@ -24,7 +24,9 @@ var cidAtlasEntry = atlas.BuildEntry(cid.Cid{}).
 
 // BigIntAtlasEntry gives a reasonable default encoding for big.Int. It is not
 // included in the entries by default.
-var BigIntAtlasEntry = atlas.BuildEntry(big.Int{}).Transform().
+var BigIntAtlasEntry = atlas.BuildEntry(big.Int{}).
+	UseTag(2). // see https://www.iana.org/assignments/cbor-tags/cbor-tags.xhtml
```

Comment:
So... I'm not sure if this should be considered a concern or not... but just to mention it in passing, in case it is relevant:

We're mostly trying to move away from acknowledging or leaning on tags in the future direction of IPLD work.  They don't map well to anything outside of CBOR, and thus aren't really good citizens to have in the Data Model as we work on formalizing that concept more explicitly.

So, in the HAMTs we introduce in the near future based on go-ipld-prime stuff, tags (aside from their use in links, which was already a moderately painful special case) will definitely be absent.  We're not allowing tags to appear above the codec layer (which is how they're acceptable for parsing links; but means they won't be allowed to be visible to higher level constructs like the advanced layouts, of which a HAMT is an instance).

But, like I said, I dunno if this is a significant consideration for this library and this diff.  And I'm not sure I've fully charted the migration story between this HAMT and the HAMTs that @rvagg is working on (though more thoughts may exist on that and I'm just not loaded on it myself).

Reply:
I guess this is ancillary to this PR anyway - and so if it's at all contentious - I'll just get rid of it.